### DISCUSSION TITLE
Adding time to "ML Surrogates for Chemical Processes with OMLT":

### DISCUSSION THREAD

**Discussion by SaM-92** [cog-imperial/OMLT#78] [Q&A] [answered]
Hello,

Thank you for this nice library. I'd want to ask a question about adding a time index to the model. Can we add time (t) to the model and then modify the input variable for each (t) and receive the forecast from NN for each (t), then maximum the objective function for the summing over 24 hours?

I put it in the equation as below. So, basically what I want to do is solving the same problem over a period of let's say 24 hours. 

Thanks so much in advance! :) 

![image](https://user-images.githubusercontent.com/78544726/168321887-1235f332-0afb-4c05-ac66-a26fd8b94834.png)

**Comment by rmisener** [comment, marked as answer]
This is the same question as Issue #79, let's keep the discussion in one place and address it there.
